board/leica/crocodile/Kconfig

```diff
@@ -11,5 +11,13 @@ config SYS_CONFIG_NAME
 
 config IMX_CONFIG
 	default "board/leica/crocodile/imximage-8mm-lpddr4.cfg"
+	
+config LEICA_FSPI_NOR_IMG_ADDR
+	hex "Address of image to be booted from NOR flash in bytes"
+	default 0x200000
+
+config LEICA_FSPI_NOR_IMG_SIZE
+	hex "Size of image to be booted from NOR flash in bytes"
+	default 0x1600000
```

Comment:
Just for clarification: where those two values are taken from? How is the `size` calculated? Can it be guaranteed that we do not overflow it?

Reply:
These two values are taken from disklayout for QSPI in meta-geosurv.
To check for "overflow" is probably very difficult, because the size of the connected NOR flash is read dynamically by the driver (during `sf probe`).